templates/travis/.travis/validate_commit_message.py.j2

```diff
@@ -51,7 +51,7 @@ print("Checking commit message for {sha}.".format(sha=sha[0:7]))
 regex = r"(?:{keywords})[\s:]+#(\d+)".format(keywords=("|").join(KEYWORDS))
 pattern = re.compile(regex)
 
-issues = pattern.findall(message)
+issues = pattern.findall(message, flags=re.IGNORECASE)
```

Comment:
GHA did not like it:
```
TypeError: 'flags' is an invalid keyword argument for findall()
```
https://github.com/pulp/plugin_template/pull/256/checks?check_run_id=1038033183#step:7:23